1_fetch/src/my_happy_script.R

```diff
@@ -20,16 +19,18 @@ library(stringr)
 library(sbtools)
 library(whisker)
 
+# create new output directory
 project_output_dir <- 'my_dir'   # 1_fetch/out/my_dir
-
 dir.create(project_output_dir)
 
-# Get the data from Science Base    # good
-getCSV <- function(){
-  mendota_file <- file.path(project_output_dir, 'model_RMSEs.csv')
+#include a parameter for the path to the output folder from tHe project directory:
+out_folder <-  "1_fetch/out/my_dir"
```

Comment:
This can just be `1_fetch/out`

Reply:
Also, you can move this down to the "instructions" section that shows you how to use the functions.